src/pretix/api/serializers/item.py

```diff
@@ -44,6 +67,18 @@ class Meta:
                   'require_voucher', 'hide_without_voucher', 'allow_cancel',
                   'min_per_order', 'max_per_order', 'checkin_attention', 'has_variations',
                   'variations', 'addons')
+        read_only_fields = ('variations', 'addons')
```

Comment:
Should we declare ``picture`` read-only for now and document that? Also, ``has_variations`` is read-only, this is a calculated property that can't be changed (and should be documented as such).

Reply:
I'll skip trying to implement image uploads for now.  So I have marked both as read-only in the table view for the endpoint. Not sure if there should be a more explicit way to document read-only fields?

Reply:
I think for now it is fine.